django_cloud_tasks/models.py

```diff
@@ -0,0 +1,122 @@
+from typing import Optional
+from django.db import transaction
+from django.apps import apps
+
+from datetime import datetime
+from django.db import models
+from django_cloud_tasks import tasks
+from django_cloud_tasks.errors import PipelineError
+
+class Pipeline(models.Model):
+    name = models.CharField(max_length=50)
+
+    def start(self):
+        routines = self.routines.filter(next_routines__id__isnull=True)
+        if not routines.exists():
+            raise PipelineError("Not enought routines to start")
+
+        if routines.count() > 1:
+            raise PipelineError("Too many routines to start at once")
```

Comment:
Why is that an issue?

Reply:
I've discussed restricting that to limit 1 routine to start for each pipeline